docs/devsite-help/support.md

```diff
@@ -0,0 +1,223 @@
+# Support
+
+The vast majority of the code in the Google Cloud Libraries is autogenerated. If
+you're running into problems, it *probably* isn't in that code (although that's
+always a possibility). The problems we see being reported usually fall into a
+few different buckets, described below.
+
+In general, after reading the advice below, if you have a support contract with
+Google, please create an issue in the
+[support console](https://cloud.google.com/support/) instead of filing on
+GitHub. This will ensure a timely response.
+## Authentication issues
+
+Authentication is a complex topic. We would advise reading the general
+[Google Cloud Authentication documentation](https://cloud.google.com/docs/authentication)
+first in order to determine what authentication strategy is most suitable for
+your application. After that, the[.NET Authentication documentation]
+(https://developers.google.com/api-client-library/dotnet/guide/aaa_overview)
+should help you implement that strategy.
+
+If either it's unclear how to implement your chosen authentication strategy, or
+you have code which you expect to work but which doesn't, it's worth trying to
+isolate that aspect from any other application code. Creating a small console
+application which initializes appropriate credentials and then uses them in a
+single API call can help to make things much clearer.
+
+## Network/environmental issues
+
+Often users report issues which turn out to be due to unusual network
+environments (in particular proxies which may not fully implement HTTP/2.0) or
+other unexpected platform. Please review the [list of supported platforms]
```

Comment:
```suggestion
other unexpected platforms. Please review the [list of supported platforms]
```

Reply:
Done. (Although it's still a slightly odd phrase; I might want to tweak it a bit later.)